### PR TITLE
fix(styled-components): Fix react-native-elements related problems

### LINKED ISSUE #615: IssueDescription is not rendered very well
It may due to #563.

Before(v1.3):

21509632125_ pic

Now:

41509632185_ pic_hd

### PR DESCRIPTION
**Pull request by machour**
Spent the afternoon fixing up the problems introduced by the styled-components migration:

- Upgrade to react-native-elements 0.18.1 which contains a fix for styled components supports
- Fixed problems:
  1. `<EntityInfo/>`: The test on the company was inverted. Also renamed the prop.
  2. `<IssueDescription/>`: Fix the repo link header, the issue title & use InlineLabel instead of LabelButton for issues/pulls label
  3. `<IssueEventListItem/>`: Fix icon styling
  4. `<*List />` : Adjusted the border bottom color, as the default changed in RNE from 0.7 to 0.17 and this is why we had bolder lines.

This should fix #615 #580 and several comments in #569

### CODE REVIEW

**Comment by machour** [comment]
We're back to normal on a lot of things <3

screen shot 2017-11-04 at 10 06 29 pm
screen shot 2017-11-04 at 10 07 05 pm

File: src/components/entity-info.component.js [hunk 1]
```diff
@@ -78,7 +82,7 @@ export const EntityInfo = ({ entity, orgs, locale, navigation }: Props) => {
             }}
             subtitle={entity.company}
             onPress={() => navigateToCompany(entity.company, orgs, navigation)}
-            unknown={companyInOrgs(entity.company, orgs)}
+            hideChevron={!companyInOrgs(entity.company, orgs)}
```
Comment (andrewda) [inline src/components/entity-info.component.js]: 🙏

**Comment by machour** [comment]
Last commit upgrades RNE to 0.18.2 that fixes another small problem with padding when using hideChevron on an item 🕵️

Before:
screen shot 2017-11-06 at 4 07 26 pm

After:
screen shot 2017-11-06 at 4 13 16 pm

**Comment by chinesedfan** [comment]
@gitpoint/maintainers Finally, I determined to merge this important PR. I have to admit that I only tested in iOS simulator by myself. But without it, there are so many bugs in our code base.

And personally, I trust @machour have tested thoroughly in both platforms, as well as @andrewda 's review. In the worst case, we can fix later. :)

**Pull request merged**